extras/mmu/mmu.py

```diff
@@ -3193,7 +3196,7 @@ def _initialize_encoder(self, dwell=False):
             self.encoder_sensor.reset_counts()
 
     def _get_encoder_dead_space(self):
-        if self.gate_homing_endstop in [self.ENDSTOP_GATE, self.ENDSTOP_POST_GATE_PREFIX]:
+        if self.has_encoder() and self.gate_homing_endstop in [self.ENDSTOP_GATE, self.ENDSTOP_POST_GATE_PREFIX]:
```

Comment:
Reviewing the body of code in the bowden auto calibration it appeared to be mostly written to work with and without an encoder. This was the only method call in there that I found where it was missing an encoder check before using data related to the encoder.